docs/source/projects.rst

```diff
@@ -1,17 +1,25 @@
 
 .. _projects:
 
-Projects using birdhouse
-========================
+Architectures examples
+======================
 
 
 .. contents::
     :local:
     :depth: 3
 
-Birdhouse is used in the following projects:
+The birdhouse :ref:`framework is modular organized <framework_structure>` to enable a flexible architecture design depending on the projects needs. Due to the OCG Standard, software components non-birdhouse components can be combined for interoperability. Here are some examples of real projects to show the flexibility and potential of the birdhouse framework.
```

Comment:
The birdhouse :ref:`framework is modular <framework_structure>` to enable a flexible architecture design based on individual projects' needs. The reliance on OCG standards allows independent software components to be combined and interoperate. Here are some examples of real projects that show the flexibility and potential of the birdhouse framework.